### PR TITLE
Switch to lib.addKeybind

### PR DESCRIPTION
**Pull request by Molicheu**
## Description

My pull request switches the keybinding method
## Checklist

- [ x ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ x ] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -1,73 +1,66 @@
...
-local harnessOn = false
+local playerState = LocalPlayer.state
 local speedMultiplier = config.useMPH and 2.237 or 3.6
 local minSpeeds = {
-    unbuckled = config.minSpeedUnbuckled / speedMultiplier,
-    buckled = config.minSpeedBuckled / speedMultiplier,
-    harness = config.harness.minSpeed  / speedMultiplier
+    unbuckled = config.minSpeedUnbuckled * speedMultiplier,
+    buckled = config.minSpeedBuckled * speedMultiplier,
+    harness = config.harness.minSpeed * speedMultiplier
```
Comment (TonybynMp4) [inline client/main.lua:9]: Not how that works, you take the value in kmh / mph and convert it to gta units, not the other way around.

**Review by TonybynMp4 — CHANGES_REQUESTED** [review]

**Review by TonybynMp4 — APPROVED** [review]
looks good to me

File: client/main.lua [hunk 2]
```diff
@@ -85,12 +78,16 @@ end)
...
     local class = GetVehicleClass(cache.vehicle)
+
     if not cache.vehicle or class == 8 or class == 13 or class == 14 then
         exports.qbx_core:Notify(locale('notify.notInCar'), 'error')
         return
     end
-    if not harnessOn then
+
+    if not playerState.harness then
         LocalPlayer.state:set('invBusy', true, true)
```
Comment (FjamZoo) [inline client/main.lua:91]: Make this playerState:set('invBusy', true, true)

File: client/main.lua [hunk 3]
```diff
@@ -103,8 +100,8 @@ RegisterNetEvent('qbx_seatbelt:client:UseHarness', function(ItemData)
             }
         }) then
             LocalPlayer.state:set('invBusy', false, true)
```
Comment (FjamZoo) [inline client/main.lua:102]: Make this playerState:set('invBusy', true, true)

**Review by FjamZoo — CHANGES_REQUESTED** [review]

**Comment by FjamZoo** [comment]
remove invBusy shit, we dont need it because we are using ox progressbar which does it

**Pull request merged**